_posts/events/2018-05-07---build-2018-live-stream.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: single
+title: "2018-05-07 - BUILD 2018 live stream"
+date: 2018-04-21 13:37:00 +0000
+comments: true
+published: true
+categories: ["events"]
+tags: ["Events"]
+alias: ["/events/2018-05-07---build-2018-live-stream"]
+author: Kris van der Mast
+redirect_from:
+ - /events/2018-05-07---build-2018-live-stream.html
+ - /events/2018/05/07/build-2018-live-stream.html
+---
+
+The developers conference we're all been waiting for is happening again in May. We're talking about Microsoft //BUILD of course. As not everyone can attend in person we thought it would be a great idea to partner with our friends of MADN, OWIC and VISUG and have a live stream. Microsoft Belgium opens up the doors for us. 
```

Comment:
`MADN, OWIC and VISUG` maybe mention full names or links in case they want to learn more?